examples/config.yml

```diff
@@ -201,16 +201,45 @@ logging:
   #lighthouse_metrics: false
 
 # Handshake Manger Settings
-#handshakes:
+handshakes:
   # Handshakes are sent to all known addresses at each interval with a linear backoff,
   # Wait try_interval after the 1st attempt, 2 * try_interval after the 2nd, etc, until the handshake is older than timeout
   # A 100ms interval with the default 10 retries will give a handshake 5.5 seconds to resolve before timing out
   #try_interval: 100ms
   #retries: 20
+
   # trigger_buffer is the size of the buffer channel for quickly sending handshakes
   # after receiving the response for lighthouse queries
   #trigger_buffer: 64
 
+  # pki can be used to mask the contents of handshakes and makes handshaking with unintended recipients more difficult
+  # all settings respond to a reload
+  psk:
+    # mode defines the how pre shared keys can be used in a handshake
+    # `none` (the default) does not send or receive using a psk. Ideally `enforced` is used
```

Comment:
I wonder how much value the `none` state provides versus using `transitional-accepting` with no PSKs as the default. This would reduce what needs to be modified in the config to enable the feature - and reduces the cognitive overhead of 4 possible states to 3.